kratos/tests/cpp_tests/utilities/test_force_and_torque_utils.cpp

```diff
@@ -0,0 +1,123 @@
+//    |  /           |
```

Comment:
I've the feeling that the Python and cpp tests are testing exactly the same. If this is the case, I'd only keep the Python one (we test the same functionalities as well as the Python export).